src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionController.kt

```diff
@@ -67,6 +71,9 @@ class IdentificationSessionsController(
         path = ["/{useIdSessionId}/$TCTOKEN_PATH_SUFFIX"],
         produces = [MediaType.APPLICATION_XML_VALUE]
     )
+    @Operation(summary = "Get TC token for this session")
+    @ApiResponse(responseCode = "200")
+    @ApiResponse(responseCode = "404", description = "No corresponding session found for that useIdSessionId", content = [Content()])
```

Comment:
All secured endpoints should also define a `SecurityScheme`: https://www.baeldung.com/spring-openapi-global-securityscheme